config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  # get 'users/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  # get "/users", to: "users#index"
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get '/users/:id', to: 'users#user'
+  get '/users/:id/posts', to: 'users#posts'
+  get 'users/:id/posts/:post_id', to: 'users#post'
```

Comment:
- [ ] The `to:` attribute on line 7 should point to `users#show`

- [ ] Kindly use the correct params. These should be:

  ```
  get '/users/:user_id/posts', to: 'posts#index'
  get '/users/:user_id/posts/:id', to: 'posts#show'
  ```

  Alternatively, you can use the `resources` helper to defined your all your routes.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```